tests/emailpassword/test_emailverify.py

```diff
@@ -166,15 +167,16 @@ async def test_the_generate_token_api_with_valid_input_email_verified_and_test_e
     cookies = extract_all_cookies(response_1)
 
     verify_token = await create_email_verification_token(user_id)
-    if verify_token.token is None:
-        raise Exception("Should never come here")
-    await verify_email_using_token(verify_token.token)
+    if isinstance(verify_token, CreateEmailVerificationTokenOkResult):
```

Comment:
Earlier we would throw an error if `if verify_token.token is None:`. Now, if that's the case, `if isinstance(verify_token, CreateEmailVerificationTokenOkResult):` will be `false`, but we don't throw an error anymore. Please make sure to not change the semantics of the tests.